src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -11,6 +11,14 @@ import CropperController from '../components/CropperController';
 const LARGE_SCREEN = 1201;
 const MIN_SIZE = 350;
 
+const layouts = {
+  choosePhoto: 1,
+  cropPhoto: 2,
+  watchUpload: 3,
+  previewPhoto: 4,
+  screenReaderError: 5
```

Comment:
Except for the error, the layouts are named as actions from the user's perspective.